src/config_parser_utils/merge_configuration_files.py

```diff
@@ -15,59 +19,105 @@
 """
 import configparser
 import logging
+from logging.handlers import SysLogHandler
+import shutil
 import sys
 import os
+import re
+
+CFG_LINE_RGX = r"^(\S+)\s*=\s*(.*)$"
+
+def setup_logger():
+    """Configures and returns a logger that sends logs to syslog."""
+    logger = logging.getLogger('upgrade')
```

Comment:
please use a more informative name
```suggestion
    logger = logging.getLogger('ufm-configurations-merger')
```

Reply:
agree, and we know the name of the plugin, it would even be better to add it to the name

Reply:
fixed